org.gnome.gitlab.cheywood.Iotas.json

```diff
@@ -0,0 +1,130 @@
+{
+    "app-id": "org.gnome.gitlab.cheywood.Iotas",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "iotas",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--share=network",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "*.a",
+        "*.la",
+        "/include",
+        "/lib/cmake",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/aclocal",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig"
+    ],
+    "modules": [
+        {
+            "name" : "gtksourceview",
```

Comment:
this version of gtksourceview is in the Sdk.

Reply:
Of course it is! Oops. :facepalm: Thanks.